docs/source/en/model_doc/bert.mdx

```diff
@@ -41,6 +41,54 @@ Tips:
 
 This model was contributed by [thomwolf](https://huggingface.co/thomwolf). The original code can be found [here](https://github.com/google-research/bert).
 
+## Resources
+
+**Text classification**
+- A blog post on [BERT Text Classification in a different language](https://www.philschmid.de/bert-text-classification-in-a-different-language).
+- A notebook for [Finetuning BERT (and friends) for multi-label text classification](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/BERT/Fine_tuning_BERT_(and_friends)_for_multi_label_text_classification.ipynb).
```

Comment:
I would perhaps add non-official example scripts/notebooks at the bottom?

Reply:
Sure thing, I'll pull some from our community resources and add an emoji or something next to it to indicate they are non-Hugging Face scripts/notebooks! 

I'll also add at the bottom a note saying you can open a PR to submit a notebook/resource (as long as its significant and isn't a duplicate of something else) to be added to the model docs. What do you think?